demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -543,8 +544,14 @@ def start_mdx_server(handle_error: Optional[Callable] = None, file_path: Optiona
         with ReadMeValidator._MDX_SERVER_LOCK:
             if not ReadMeValidator._MDX_SERVER_PROCESS:
                 mdx_parse_server = Path(__file__).parent.parent / 'mdx-parse-server.js'
-                ReadMeValidator._MDX_SERVER_PROCESS = subprocess.Popen(['node', str(mdx_parse_server)],
-                                                                       stdout=subprocess.PIPE, text=True)
+                try:
+                    ReadMeValidator._MDX_SERVER_PROCESS = subprocess.Popen(['node', str(mdx_parse_server)],
+                                                                           stdout=subprocess.PIPE, text=True)
+                except FileNotFoundError:
+                    error_message = Errors.error_uninstall_node()
+                    print_error(error_message)
+                    sys.exit(1)
```

Comment:
`exit(1)` is a messy exit, can we use an exception to stop the running? See `BlockingValidationFailureException` as an example.